package.json

```diff
@@ -2,13 +2,13 @@
   "name": "ulabel",
   "description": "An image annotation tool.",
   "version": "0.6.10",
-  "main": "index.js",
-  "module": "src/index.js",
+  "main": "dist/ulabel.js",
+  "module": "dist/ulabel.js",
```

Comment:
does it make sense to make one/both of these the minified version? tbh I'm not sure if there's any difference btwn `main` and `module`

Reply:
No, I think the un-minified version should remain the default. I personally am not super familiar with node packages work tbh but this should explain the difference between main and module https://stackoverflow.com/questions/42708484/what-is-the-module-package-json-field-for